dockers/for_rocm/Dockerfile

```diff
@@ -0,0 +1,26 @@
+
+FROM rocm/pytorch:latest
+
+RUN \
+    pip install click && \
+    pip install deepdiff && \
+    pip install fastapi && \
+    pip install lightning-cloud && \
+    pip install -U Jinja2 && \
+    pip install starsessions && \ 
+    pip uninstall fsspec && \ 
+    pip install fsspec==2022.1.0 && \ 
+    pip install jsonargparse && \
+    pip install -U jsonargparse[signatures]
```

Comment:
why not install it as a single cmd so that eventually collisions are resolved?

Reply:
taken care of in the newest commit